fava/static/javascript/charts.js

```diff
@@ -152,7 +152,7 @@ class TreeMapChart extends BaseChart {
 
     if (this.selections.cells.empty()) {
       this.selections.empty = this.canvas.append('text')
-        .text('Chart is empty.');
+        .text(window.favaTranslations.chartEmpty);
```

Comment:
There should be some helper function, `_` maybe, so that this is just `_('Chart is empty.')`